jib-core/src/main/java/com/google/cloud/tools/jib/registry/RegistryEndpointCaller.java

```diff
@@ -164,15 +164,6 @@ private T call(URL url) throws IOException, RegistryException {
             throw new RegistryUnauthorizedException(serverUrl, imageName, responseException);
           }
 
```

Comment:
Is it our job to tell the user of a permanent redirect? For example would it make sense to do:
```
[WARN] "xyz is permanently redirected to abc, please update your config?"
```

Reply:
I don't think this is necessary or makes sense to show to users. Users would mostly use the right registry URL and the image reference at the surface. If 308 is returned, it would most likely be that the entity initiating a request is some server. After all, this is very similar to 301 (Moved Permanently). And usually you just follow redirects, understandably.